### PR TITLE
COM-2003 Added French text fixed few text issues

### PR DESCRIPTION
**Pull request by hrishid**
COM-2003 Changed Template for gateway-Giftcard-created

COM-2003 email template for gatewayGiftCard

### CODE REVIEW

File: labels/en-US.json [hunk 1]
```diff
@@ -239,6 +239,10 @@
   "friday": "Friday",
   "from": "from",
   "fullDesc": "Description",
+  "gatewayGiftCardDeleteEmail": "DO NOT DELETE THIS EMAIL",
+  "gatewayGiftCardEmailIntro": "Thank you for purchasing a {0} Gift Card. Do not delete this message -- to redeem the gift card you must use the gift card claim code below. You may want to print a copy of this message for easy reference later.",
+  "gatewayGiftCardNeedsCard": " You will need the card to complete your purchase" ,
```
Comment (ghost) [inline labels/en-US.json]: Text doesn't match what's in the jira issue. Should say `card number`.

File: labels/de-DE.json [hunk 2]
```diff
@@ -181,6 +181,10 @@
   "freeShipping": "Versand kostenfrei",
   "from": "aus",
   "fullDesc": "Beschreibung",
+  "gatewayGiftCardDeleteEmail":"NE SUPPRIMEZ PAS CET EMAIL",
+  "gatewayGiftCardEmailIntro":"Merci d'avoir acheté une {0} carte-cadeau. Ne supprimez pas ce message - pour utiliser la carte-cadeau, vous devez utiliser le code de réclamation de la carte-cadeau ci-dessous. Vous souhaiterez peut-être imprimer une copie de ce message pour une référence ultérieure.",
+  "gatewayGiftCardNeedsCard": " Vous aurez besoin de la carte pour finaliser votre achat",
```
Comment (ghost) [inline labels/de-DE.json]: Verify text is correct per english.

**Review by ghost — CHANGES_REQUESTED** [review]
Looks good but double check the text. See my comments. Also please reword your commit message to be more clear.

File: labels/en-US.json (hunk 1 above)
Comment (hrishid) [reply labels/en-US.json]: Changed the text and squashed commits that's why its saying outdated.

**Review by ghost — APPROVED** [review]

**Pull request merged**